psi4/src/psi4/libscf_solver/frac.cc

```diff
@@ -112,15 +115,8 @@ void HF::frac() {
                     nbeta_++;
             }
 
-            // Throw if the user is insane
             if (val < 0.0)
-                throw PSIEXCEPTION(
-                    "Fractional Occupation SCF: Psi4 is not configured for positrons. Please annihilate and start "
-                    "again");
-            if (val == 0.0)
-                throw PSIEXCEPTION(
```

Comment:
Is the `val == 0.0` check not needed anymore?

Reply:
Correct. As I said in the long description, "This PR allows frac to treat zero occupation number", which it was able to do (albeit incorrectly) pre-#2280.